### PR TITLE
CI Update - .NET 7 Preview 4 assemblies

### PR DESCRIPTION
**Pull request by gewarren**
Build.Reason:Manual by Genevieve Warren
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=296273&view=results
 source_repo.branch:net7-pre4
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

### CODE REVIEW

**Comment by gewarren** [comment]
cc @carlossanlop

File: xml/System.Numerics/IAdditiveIdentity`2.xml [hunk 1]
```diff
@@ -47,7 +41,7 @@
         <ReturnType>TResult</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the additive identity of the current type.</summary>
+        <summary>To be added.</summary>
```
Comment (gewarren) [inline xml/System.Numerics/IAdditiveIdentity`2.xml:44]: And here.

File: xml/System.Numerics/IBitwiseOperators`3.xml [hunk 2]
```diff
@@ -12,31 +12,25 @@
...
-      <AttributeName Language="C#">[System.Runtime.Versioning.RequiresPreviewFeatures("Generic Math is in preview.", Url="https://aka.ms/dotnet-warnings/generic-math-preview")]</AttributeName>
-      <AttributeName Language="F#">[&lt;System.Runtime.Versioning.RequiresPreviewFeatures("Generic Math is in preview.", Url="https://aka.ms/dotnet-warnings/generic-math-preview")&gt;]</AttributeName>
-    </Attribute>
-  </Attributes>
   <Docs>
-    <typeparam name="TSelf">The type that implements this interface.</typeparam>
-    <typeparam name="TOther">The type that will is used in the operation with <typeparamref name="TSelf" />.</typeparam>
-    <typeparam name="TResult">The type that contains the result of <typeparamref name="TSelf" /> op <typeparamref name="TOther" />.</typeparam>
-    <summary>Defines a mechanism for performing bitwise operations over two values.</summary>
+    <typeparam name="TSelf">To be added.</typeparam>
```
Comment (gewarren) [inline xml/System.Numerics/IBitwiseOperators`3.xml:23]: Comment loss. Looks like we'll have to manually port the doc comments for the renamed interfaces.

File: xml/System/IParsable`1.xml [hunk 3]
```diff
@@ -1,40 +1,34 @@
...
-  <Attributes>
-    <Attribute>
-      <AttributeName Language="C#">[System.Runtime.Versioning.RequiresPreviewFeatures("Generic Math is in preview.", Url="https://aka.ms/dotnet-warnings/generic-math-preview")]</AttributeName>
-      <AttributeName Language="F#">[&lt;System.Runtime.Versioning.RequiresPreviewFeatures("Generic Math is in preview.", Url="https://aka.ms/dotnet-warnings/generic-math-preview")&gt;]</AttributeName>
-    </Attribute>
-  </Attributes>
   <Docs>
-    <typeparam name="TSelf">The type that implements this interface.</typeparam>
-    <summary>Defines a mechanism for parsing a string to a value.</summary>
+    <typeparam name="TSelf">To be added.</typeparam>
```
Comment (gewarren) [inline xml/System/IParsable`1.xml:21]: Comment loss.

**Review by gewarren — COMMENTED** [review]
For the renamed types, there's some data loss in the documentation comments that we can address in a separate PR.

File: xml/System.Numerics/IAdditiveIdentity`2.xml [hunk 4]
```diff
@@ -12,31 +12,25 @@
...
-    <Attribute>
-      <AttributeName Language="C#">[System.Runtime.Versioning.RequiresPreviewFeatures("Generic Math is in preview.", Url="https://aka.ms/dotnet-warnings/generic-math-preview")]</AttributeName>
-      <AttributeName Language="F#">[&lt;System.Runtime.Versioning.RequiresPreviewFeatures("Generic Math is in preview.", Url="https://aka.ms/dotnet-warnings/generic-math-preview")&gt;]</AttributeName>
-    </Attribute>
-  </Attributes>
   <Docs>
-    <typeparam name="TSelf">The type that implements this interface.</typeparam>
-    <typeparam name="TResult">The type that contains the additive identify of <typeparamref name="TSelf" />.</typeparam>
-    <summary>Defines a mechanism for getting the additive identity of a given type.</summary>
+    <typeparam name="TSelf">To be added.</typeparam>
```
[the remark this answers is not in the document]
Comment (carlossanlop) [reply xml/System.Numerics/IAdditiveIdentity`2.xml:22]: I can take care of this with the porting tool.

**Review by carlossanlop — APPROVED** [review]
LGTM @gewarren . I'll make sure the porting tool addresses the renamed API documentation.

**Pull request merged**